src/Components/SharedComponents/AuthBackGround/AuthBackGround.tsx

```diff
@@ -0,0 +1,92 @@
+import { useLocation } from "react-router-dom";
+import { Box, Typography } from "@mui/material";
+import background1 from "../../../assets/images/bg1.png";
```

Comment:
We can have an index.ts file in the images folder and include all the images in it and then re-import it when needed. This will make all of our imports from a single file, we will also be able to get auto completion for the imports instead of needing to write the path manually.

Reply:
done